Sources/SKSupport/Platform.swift

```diff
@@ -10,10 +10,74 @@
 //
 //===----------------------------------------------------------------------===//
 
-import enum TSCUtility.Platform
+import Foundation
+
+import class TSCBasic.Process
+import struct TSCBasic.AbsolutePath
+import var TSCBasic.localFileSystem
+
+private func isAndroid() -> Bool {
+  return (try? localFileSystem.isFile(AbsolutePath(validating: "/system/bin/toolchain"))) ?? false ||
+      (try? localFileSystem.isFile(AbsolutePath(validating: "/system/bin/toybox"))) ?? false
+}
+
+private func isDebian() -> Bool {
+  return (try? localFileSystem.isFile(AbsolutePath(validating: "/etc/debian_version"))) ?? false
+}
+
+private func isFedora() -> Bool {
+  if (try? localFileSystem.isFile(AbsolutePath(validating: "/etc/redhat-release"))) ?? false {
+    return true
+  }
+
+  if (try? localFileSystem.isFile(AbsolutePath(validating: "/etc/centos-release"))) ?? false {
+    return true
+  }
+
+  if (try? localFileSystem.isFile(AbsolutePath(validating: "/etc/fedora-release"))) ?? false {
+    return true
+  }
+
+  return (try? localFileSystem.readFileContents(AbsolutePath(validating: "/etc/system-release")))?.cString.hasPrefix("Amazon Linux release 2") ?? false
+}
+
+public enum Platform: Equatable {
+  public enum LinuxDistribution: Equatable {
+    case debian
+    case fedora
+  }
```

Comment:
I don’t think we are differentiating between Debian and Fedora anywhere in SourceKit-LSP (and I don’t see us needing the differentiation anytime soon), so maybe we should just remove the distinction, especially since the Debian vs Fedora check seems to be the most complicated.

Reply:
I ... uh, yeah, I should've done that.  I don't know why I didn't.  Sorry.